### PR TITLE
Chapter3.16字误、排版微调

### PR DESCRIPTION
**Pull request by yongboy**
修改内容如下：
* 3.15章增加图3.8图片显示，方便理解
* 3.16字误修改，内容安排更合理化一些

:))

### CODE REVIEW

**Comment by yongboy** [comment]
追加 `ffeb8ec` 提交内容：

- `torch.nn.MSELoss()` 计算规则为 `(input - target) ** 2`，因此不需要像 `mxnet.gluon.loss.L2Loss()` 其结果再 `*2`
- 我把3.16章`模型选择`输出结果替换成 `torch.nn.MSELoss()`运算结果不再乘以2后的真实输出
  - 这样修改之后，和原版相比在输出结果上比较一致了

 :))
